src/main/resources/templates/casUnsupportedInstitutionLoginView.html

```diff
@@ -0,0 +1,107 @@
+                        </tr>
+                        </tbody>
+                    </table>
+                </section>
+
+                <section class="card-message">
+                    <h2 th:utext="#{screen.instnloginna.heading}"></h2>
+                    <p th:utext="#{screen.instnloginna.message(${osfUrl.institutionsHome})}"></p>
+                </section>
+
+                <section class="card-message">
+                    <h3 th:utext="#{screen.instnloginna.existing.heading}"></h3>
+                    <p th:utext="#{screen.instnloginna.existing.message}"></p>
+                </section>
+
+                <section class="form-button">
+                    <a class="mdc-button mdc-button--raised button-osf-blue" th:href="@{/login(service=${service?.originalUrl ?: ''})}">
+                        <span class="mdc-button__label" th:utext="#{screen.instnloginna.existing.button}"></span>
+                    </a>
+                </section>
+
+                <section id="osfUnsupportedInstitutionLogin">
+                    <div class="card-message">
+                        <p th:utext="#{screen.instnloginna.existing.setpw.message}"></p>
+                    </div>
+                </section>
+
+                <div class="form-wrapper">
+                    <form method="post" id="fm1" th:object="${credential}" th:action="@{${osfUrl.forgotPasswordInstitution}}">
+                        <section class="cas-field cas-field-primary my-3">
+                            <div class="mdc-text-field  d-flex">
+                                <input class="mdc-text-field__input"
+                                       id="username"
+                                       size="256"
+                                       type="email"
+                                       name="forgot_password-email"
+                                       th:accesskey="#{screen.instnloginna.existing.setpw.label.email.accesskey}"
+                                       autocomplete="off" />
+                                <label for="username" class="mdc-floating-label" th:utext="#{screen.instnloginna.existing.setpw.label.email}"></label>
+                            </div>
+                        </section>
+                        <section>
+                            <div th:replace="fragments/submitbutton :: submitButton (buttonDisabled=false, buttonCustomization='button-osf-blue', messageKey='screen.instnloginna.existing.setpw.button')" />
+                        </section>
+                    </form>
+                </div>
+
+                <hr class="my-4" />
+                <section class="text-with-mdi">
+                    <i class="mdi mdi-account-question-outline mdi-before-text"></i>
+                    <span>
+                            <a href="https://help.osf.io/hc/en-us/articles/360019737194-Sign-in-to-OSF" th:utext="#{screen.generic.link.support}"></a>
+                        </span>
+                </section>
```

Comment:
Replaced the "hidden" and deprecated `pwlinks` fragment with a "footer" similar to the other institution page.